lib/ossert/fetch/github.rb

```diff
@@ -123,7 +123,7 @@ def date_from_tag(sha)
         tag_info = tag_info(sha)
         return tag_info[:tagger][:date] if tag_info
         value = commit(sha)[:commit][:committer][:date]
-        DateTime.new(*value.split('-').map(&:to_i)).to_i
+        Date.new(*value.split('-').map(&:to_i)).to_time(:utc).to_i
```

Comment:
Sorry, but here I have some concerns.
I hope that some day we'll get rid of the ActiveSupport.
`#to_time(:utc)` is unfortunately - is a part of it...
Let's try to find other way to fix.
WDYT?

Reply:
Ok, i can try.